### ISSUE TITLE
Get code into this repo

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/bpp-pass2-ui#2]
At the moment, this is an empty repo with issues only but no code.  MS Access isn't an entirely natural fit for version management since it's so UI-based, but I have found something on the interwebs about a way to get it to spit out code that can be tracked...  So we should probably do that at some point.  Low priority for now.

**Comment by johnklemke** [comment]
Yeah, I've made only glancing efforts at providing version and revision info, and kinda littered gdelt with copies.

**Comment by davidskalinder** [comment]
Looping @olderwoman and @limchaeyoon in here in case they want to weigh in on resource usage here...

As discussed in the team meeting yesterday, the priority of this issue increaseth, since otherwise it's not easy to have me (or anybody else) do much development concurrently with @johnklemke.

So, it does appear that putting the Access stuff under version control with git will have some gnarls to work through.  Access provides two VBA commands, `Application.SaveAsText` and `Application.LoadFromText`, that allow Access files to be exported to or imported from text files, but those commands are undocumented(!) and require supporting code; some existing tools attempt to provide this.  I think these are the main options:

1. Either of two commercial tools, [OASIS-SVN](https://dev2dev.de/index.php?option=com_content&view=article&id=44&Itemid=260&lang=en) or [Ivercy](https://www.ivercy.com/Home/Features)
1. An open-source solution, the snappily-named [msaccess-vcs-integration](https://github.com/msaccess-vcs-integration/msaccess-vcs-integration)
1. Roll and maintain our own solution (like [this](https://stackoverflow.com/questions/187506/how-do-you-use-version-control-with-access-development) or perhaps [this](http://www.accessmvp.com/Arvin/DocDatabase.txt))
1. Give up.  Carefully check via email every time two devs are working on the DB at the same time that the work won't conflict, and expect things to go wrong every so often.

(In addition to the links above, much of this info is from these two [good](https://accessexperts.com/blog/2019/07/02/using-oasis-svn-and-git-for-access-source-code-control/) [guides](https://codekabinett.com/rdumps.php?Lang=2&targetDoc=choose-source-code-control-microsoft-access).)

The commercial tools would presumably be easy to use; they're somewhat [reasonably](https://dev2dev.de/index.php?option=com_content&view=article&id=44&Itemid=260&lang=en) [priced](https://www.ivercy.com/Home/Pricing) for single users and somewhat less reasonably priced for multiple users.  I have no idea whether they do academic deals or how they would have to be installed on Winstat or whether that's even possible: we'd likely have to consult with SSCC on much of this.

msaccess-vcs-integration seems lightly maintained at this point (it looks like it's accepting external patches and bugfixes but not much else).  I'm guessing it would be at least slightly cranky to install and use (the first line of the readme is "This tool can delete / break things, take a backup before getting started.").  But of course it has all the usual advantages of open-source stuff (cost, transparency, fixability, etc.).  Written in what looks to me like VB.  I think it installs as an Access add-in, and it occurs to me that I don't know whether we'll have to consult with SSCC on that too...

Rolling our own sounds like a PITA to me -- plenty of gotchas with character encoding and including certain features in the export and so on.  But OTOH, it's not impossible that it'll turn out to be a way for us to cut the clutter of an external tool?

I think it's worth me doing at least an initial attempt to get msaccess-vcs-integration working.  If it becomes clear that it'll be a huge time sink then we can try one of the alternates?...
